test/kongintegration/parser_golden_tests_outputs_test.go

```diff
@@ -24,6 +24,11 @@ import (
 func TestParsersGoldenTestsOutputs(t *testing.T) {
 	t.Parallel()
 
+	const (
+		timeout = 5 * time.Second
+		tick    = time.Millisecond * 100
```

Comment:
```suggestion
		tick    = 100* time.Millisecond
```
lie in natural language number before unit